<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotDialog</class>
 <widget class="QDialog" name="plotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>370</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>数据显示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/plotView.png</normaloff>:/images/plotView.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>数据显示</string>
   </property>
   <widget class="QCheckBox" name="checkBox_hide_grid">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>隐藏网格</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>75</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通道A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通道B</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>水平坐标单位</string>
   </property>
   <widget class="QRadioButton" name="radioButton_countNum">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计数序号</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_echoDistance">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>回波距离m</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>320</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
